tests/integration/suite_test.go

```diff
@@ -112,20 +116,21 @@ var _ = AfterSuite(func() {
 })
 
 var _ = BeforeEach(func() {
-	testNamespace = createTestNamespace(context.Background())
+	schemaVersion = "1.0.1"
 }, 5)
 
 var _ = AfterEach(func() {
 	if _, ok := os.LookupEnv("KEEP_TESTS"); ok {
 		return
 	}
-	deleteNamespace(testNamespace)
+	//TODO: don't reuse namespace for all tests move this to suite/magefile cleanup
+	//deleteNamespace(testNamespace)
```

Comment:
What was the problem with the original code that made you comment it out? Let's get this TODO addressed in this PR so that we don't lose track of it.

Reply:
The goal was to make test execution and cleanup more resilient.  Namespace per test was cleaner during feature development.  It also helps support parallel execution of tests.  The commented out code and TODO should have been removed.  We could also remove the KEEP_TEST flag at this point, because the cleanup always happens at the start of the next run.  Let me know what you think.

Reply:
Does each test get a new namespace to run under? I'm wondering how you are running tests in parallel and also cleaning up previous test runs if they are generated names?

Reply:
Yes, each test gets its own namespace (still labeling with `porter.sh/testdata`).  The TestIntegration target removes all namespaces with the label at the start of a run.